### PR TITLE
Add missing options to the readme.

### PR DESCRIPTION
**Pull request by rhoboat**

### CODE REVIEW

**Review by bwhaley — APPROVED** [review]
LGTM!

File: README.md [hunk 1]
```diff
@@ -51,7 +51,10 @@ Option                      | Required | Description
 `--binary-sha512-checksum`  | No       | The SHA512 checksum of the binary specified by `--binary-name`. Should be exactly 128 characters..
 `--module-param`            | No       | A key-value pair of the format `key=value` you wish to pass to the module as a parameter. May be used multiple times. <br>Note: a `--` will automatically be appended to the `key` when your module is invoked<br>See the documentation for each module to find out what parameters it accepts.
 `--download-dir`            | No       | The directory to which the module will be downloaded and from which it will be installed.
-`--branch      `            | No       | Download the latest commit from this branch in --repo. This is an alternative to --tag,<br>and is used only for testing. This value is exposed to module install scripts as GRUNTWORK_INSTALL_BRANCH.
+`--binary-install-dir`      | No       | The directory to which the binary will be installed. Only applies to binaries (not modules). Default: $DEFAULT_BIN_DIR"
```
Comment (brikis98) [inline README.md:54]: This `$DEFAULT_BIN_DIR` doesn't seem to be rendering correctly in Markdown 😁

**Review by brikis98 — COMMENTED** [review]
Whoops, not sure how we missed adding these to the docs. Thx for fixing this.

File: README.md (hunk 1 above)
Comment (rhoboat) [reply README.md:54]: Oops! Good catch.

**Pull request merged**